benchmark/matrix_statistics/matrix_statistics.cpp

```diff
@@ -41,33 +41,13 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 
 #include "benchmark/utils/general.hpp"
+#include "benchmark/utils/spmv_common.hpp"
 
 
 // some Ginkgo shortcuts
 using etype = double;
 
 
-// input validation
-void print_config_error_and_exit()
-{
-    std::cerr << "Input has to be a JSON array of matrix configurations:\n"
-              << "  [\n"
-              << "    { \"filename\": \"my_file.mtx\"},\n"
-              << "    { \"filename\": \"my_file2.mtx\"}\n"
-              << "  ]" << std::endl;
-    std::exit(1);
-}
-
-
-void validate_option_object(const rapidjson::Value &value)
-{
-    if (!value.IsObject() || !value.HasMember("filename") ||
-        !value["filename"].IsString()) {
-        print_config_error_and_exit();
-    }
-}
-
-
 void initialize_argument_parsing(int *argc, char **argv[])
 {
```

Comment:
As far as I can tell, you are no longer using this function, as it is replaced by
```c++
initialize_argument_parsing(int *argc, char **argv[], std::string &header, std::string &format)
```
from ` benchmark/utils/general.hpp`.